shells/firefox/build.js

```diff
@@ -25,13 +25,13 @@ const main = async () => {
   console.log(chalk.gray('\n# Go to the unpacked directory:'));
   console.log(`cd ${unpackedPath}`);
   console.log(chalk.gray('\n# And launch Firefox with the extension enabled:'));
-  console.log('web-ext run');
+  console.log('yarn run start:firefox');
```

Comment:
Yeah, that works.  Keeps it simpler. :)

I also discovered that I can use environment variables as well so I could keep the test.js file and just offer options like this `WEB_EXT_FIREFOX=nightly yarn run test:firefox`